pressurecooker/subtitles.py

```diff
@@ -54,6 +55,9 @@ def read(self, caption_str):
                 return self.reader.read(caption_str, lang=LANGUAGE_CODE_UNKNOWN)
 
             return self.reader.read(caption_str)
+        except CaptionReadError:
```

Comment:
This looks good, but I'm wondering if this case should just immediately raise an `InvalidSubtitleFormatError` here. The `UnicodeDecodeError` could occur during detection, so returning `None` would make sense there. For this error, it seems like it would occur after detection, so halting the whole detect format and read loop seems appropriate. I could also see refactoring this to split it up into two `try`/`except` blocks for detect and read, if that also makes this clearer for the future.

Reply:
I'd also like to offer to take over making those changes if this short term fix is needed ASAP. I welcome your thoughts and contributions here though!

Reply:
Just chatted with @ivanistheone, I will go ahead make those changes so this gets pushed through.